react.md

```diff
@@ -0,0 +1,242 @@
+# Trunk Club React Styleguide
+
+This guide assumes that you are writing in the coffeescript-flavor of [JSX](https://github.com/jsdf/coffee-react-transform). It also assumes that you have a cursory understanding of what JSX is and why React uses it. See the [docs](http://facebook.github.io/react/docs/jsx-in-depth.html) for more details.
+
+## Structure of React Components
+You should structure your React Components like this to ensure consistency. Whether you're using ES6 or Coffeescript or PoopyScript, it should be ordered like this
+
+1. Attributes
+  1. If you are using a mixin that requires you to specify an attribute on the Component's prototype (and not on the instance itself), do that here as well
+
+  ```Coffeescript
+  ComponentName = React.createClass
+    displayName: 'ComponentName'
+    mixins: []
+    statics: {}
+    propTypes: {}
+    listen: {} # Custom attribute needed by a mixin
+  ```
+
+2. React lifecycle hooks
+
+  ```Coffeescript
+  getInitialState: -> {}
```

Comment:
Nope; I would look to the react docs to be the source of truth for these methods.